src/js/themes/base.js

```diff
@@ -982,6 +982,12 @@ export const generate = (baseSpacing = 24, scale = 6) => {
     menu: {
       // background: undefined,
       // extend: undefined,
+      drop: {
+        align: {
+          top: 'top',
+          left: 'left',
+        },
+      },
```

Comment:
I was starting to ask "How will this affect existing implementations? Do we want this undefined and let users opt-in?", but now see that these were previously defaultProps on Menu.js and we are simply moving where the defaults are applied, correct?

Reply:
Yes exactly. Previously, this same default was defined as Menu.defaultProps `dropAlign`. With the adjustment of the logic to fall back to the theme, I've just moved the default into the theme. No changes will be experienced on existing applications